src/cloud/cdn/configure-fastly-customize-cache.md

```diff
@@ -98,6 +98,9 @@ To set up GeoIp handling:
 
 1. After page reload, click **Upload VCL to Fastly** in the *Fastly Configuration* section.
 
+{:.bs-callout-info}
+The current Adobe Commerce Fastly GeoIP module implementation does not support redirections between websites.
```

Comment:
Does this need to specify what kind of websites? Would multiple make sense?

```suggestion
The current Adobe Commerce Fastly GeoIP module implementation does not support redirections between multiple websites.
```

Reply:
yes it make sense